recipes/minizip-ng/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class MinizipNgConan(ConanFile):
+    name = "minizip-ng"
+    description = "Fork of the popular zip manipulation library found in the zlib distribution."
+    topics = ("conan", "compression", "zip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zlib-ng/minizip-ng"
+    license = "Zlib"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package", "pkg_config"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package", "pkg_config"
    provides = "minizip"
```

I think both `minizip` and `minizip-ng` can not be used together. We need to update `minizip` too.

Reply:
Thats only true if the compatability option is True https://github.com/zlib-ng/minizip-ng/blob/6f17da582d2327c8cdc43622ddd31a185b6ef5fb/CMakeLists.txt#L604

Reply:
`minizip` and `minizip-ng` produces both the same binaries (except on Windows) but doesn't provide the same API unless compatability option is true.

Reply:
So it can be added on configure method.